src/main/java/com/snowflake/kafka/connector/internal/streaming/DirectTopicPartitionChannel.java

```diff
@@ -0,0 +1,90 @@
+package com.snowflake.kafka.connector.internal.streaming;
+
+import com.snowflake.kafka.connector.internal.streaming.channel.TopicPartitionChannel;
+import com.snowflake.kafka.connector.internal.streaming.telemetry.SnowflakeTelemetryChannelStatus;
+import com.snowflake.kafka.connector.internal.telemetry.SnowflakeTelemetryService;
+import net.snowflake.ingest.streaming.SnowflakeStreamingIngestChannel;
+import org.apache.kafka.connect.sink.SinkRecord;
+
+import java.util.concurrent.CompletableFuture;
+
+public class DirectTopicPartitionChannel implements TopicPartitionChannel {
+  @Override
+  public long getOffsetPersistedInSnowflake() {
+    return 0;
```

Comment:
this probably doesn't hurt, as it isn't used anywhere, but i tent do throw UnsupportedOperationException for such initial stub methods.

Reply:
let me leave it as it is now, it will be replaced in the next iterations